.github/workflows/haskell.yml

```diff
@@ -22,30 +22,32 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.2.8", "9.4.8", "9.6.5", "9.8.2", "9.10.1"]
-        cabal: ["3.10.3.0"]
-        os: [ubuntu-latest] # ubuntu-latest = ubuntu-22.04
-        liburing: ["liburing-2.6"]
+        ghc: ["8.10.7", "9.2.8", "9.4.8", "9.6.6", "9.8.4", "9.10.1"]
+        cabal: ["3.12.1.0"]
+        os: [ubuntu-latest] # ubuntu-latest = ubuntu-22.04 (soon to be 24.04)
+        liburing: ["system"]  # system liburing is 2.1 on 22.04, 2.5 on 24.04
```

Comment:
My intent here was to run most of our CI with the newest version, and I recall I did that because presumably it would be newer versions are "better": more (bug) fixes and maybe some optimisations. I think testing with `system` might also be fine, because I'd assume most people install through `apt-get` on `ubuntu`, but we already have `include`s that test the `system` one. What do you think?

Reply:
I don't think it makes a big difference what is the default as long as every option is covered. A tiny argument in favor of  using `system` might be that installing the binary package requires less CI resources than compiling from source.